### PR TITLE
#comment Api to make outgoing call with ozonetel

### PR DESCRIPTION
**Pull request by vyadav12**

### CODE REVIEW

File: lib/ozonetel/client.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+require 'httparty'
+
+module Ozonetel
+  class Client
+    include HTTParty
+    MANUAL_DIAL_URL = "http://cloudagent.in/CAServices/AgentManualDial.php"
+
+    def initialize(customer, api_key, campaign_name)
+      @user_name = customer
+      @api_key = api_key
```
Comment (urajat) [inline lib/ozonetel/client.rb:10]: Shouldn't api_key be the first parameter? Or is this API key specific to a customer?

File: lib/ozonetel/client.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    def agent_manual_dial(agent_id, customer_number)
+      data = {
+        'username' => @user_name,
+        'api_key' => @api_key,
+        'agentID' => agent_id,
+        'customerNumber' => customer_number,
+        'campaignName' => @campaign_name
+      }
+      call_response = HTTParty.post(MANUAL_DIAL_URL, :body => data)
```
Comment (urajat) [inline lib/ozonetel/client.rb:22]: Shouldn't this be self.post given that we have included HTTParty?

File: ozonetel.gemspec [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'ozonetel/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "ozonetel"
+  spec.version       = Ozonetel::VERSION
+  spec.authors       = ["vijay yadav"]
+  spec.email         = ["vijaynitt12@yahoo.in"]
+  spec.summary       = %q{Rest APIs to intract with ozonetel.}
```
Comment (urajat) [inline ozonetel.gemspec]: typo - "interact"

File: lib/ozonetel/client.rb (hunk 1 above)
Comment (vyadav12) [inline lib/ozonetel/client.rb:10]: Yeah, API key is specific to a customer. Here customer is urbanladder

File: lib/ozonetel/client.rb (hunk 2 above)
Comment (vyadav12) [inline lib/ozonetel/client.rb:22]: I guess its same. Or is there any difference ? If we write 'HTTParty.post' then its easy to understand for beginners.

**Pull request merged**